src/renderer/containers/ReplayFileStats/HomeRunProfile.tsx

```diff
@@ -0,0 +1,26 @@
+import Warning from "@mui/icons-material/Warning";
+import type { FileResult } from "@replays/types";
+import type { StatsType } from "@slippi/slippi-js";
+import _ from "lodash";
```

Comment:
```suggestion
```
Why is this imported but not used?

Reply:
for these two it looks like eslint doesn't actually catch the unused import because it's just an underscore - thanks for catching this

`
"@typescript-eslint/no-unused-vars": [
  "error",
  {
    argsIgnorePattern: "^_",
    varsIgnorePattern: "^_",
  },
],
`